### PR TITLE
Changed order in dropdown and initial state.

### PR DESCRIPTION
**Pull request by whitingjr**
fixes issue #573 

## Changes proposed

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.

### CODE REVIEW

File: webapp/src/domain/runs/JsonPathSearchToolbar.tsx [hunk 1]
```diff
@@ -137,15 +137,15 @@ export default function JsonPathSearchToolbar(props: ToolbarProps) {
...
-                                </DropdownItem>,
                                 <DropdownItem id="jsonb_path_query_first" key="jsonb_path_query_first">
                                     jsonb_path_query_first
                                 </DropdownItem>,
                                 <DropdownItem id="jsonb_path_query_array" key="jsonb_path_query_array">
                                     jsonb_path_query_array
                                 </DropdownItem>,
+                                <DropdownItem id="js" key="query">
+                                    js
+                                </DropdownItem>
```
Comment (shivam-sharma7) [inline webapp/src/domain/runs/JsonPathSearchToolbar.tsx:148]:
```suggestion
                                <DropdownItem id="js" key="query">
                                    js
                                </DropdownItem>
```

Comment (shivam-sharma7) [reply webapp/src/domain/runs/JsonPathSearchToolbar.tsx:148]: @whitingjr why we are showing only js text?

Comment (whitingjr) [reply webapp/src/domain/runs/JsonPathSearchToolbar.tsx:148]: @shivam-sharma7  What I've done is relocate these lines of code. When you look at the context the [dropdownItems](https://github.com/whitingjr/Horreum/blob/9c1656608f068fbac27784aeb406be77d9392f4f/webapp/src/domain/runs/JsonPathSearchToolbar.tsx#L139-L148) still has 3 items.

**Comment by whitingjr** [comment]
@shivam-sharma7 hi, I asked john to also do a review. He was involved in the initial discussion and may have an opinion on this PR. If you have anything else to query please raise it.

**Review by shivam-sharma7 — APPROVED** [review]
@whitingjr LGTM

**Comment by jesperpedersen** [comment]
Rebased, and merged.

Thanks for your contribution !

**Pull request closed without merging**